README.md

```diff
@@ -1,13 +1,15 @@
 # Portfolio
 
-> In this project I created a pertfolio which is adaptable on Desktop as well as on mobile/smartphone. My Portfolio website help others understand who i am and what technologies i know and how to get in touch with me. Please view my portfolio page in your browser by following this link https://hadi111jafari.github.io/Portfolio/
+> In this project I created a pertfolio which is adaptable on Desktop as well as on mobile/smartphone. My Portfolio website help others understand who i am and what technologies i know and how to get in touch with me. 
 
 
 ## Built With
-
 - html
 - css
 
+## Live demo 
+[Live page](https://github.com/Hadi111jafari/Portfolio)
+
```

Comment:
- [ ] Good job deploying your project but the link provided here is to your GitHub repository and not the live page. Use this link `https://hadi111jafari.github.io/Portfolio/` instead for the live page.